vtex/admin-ui
# Feat filter searchable single select

#### What is the purpose of this pull request?
Add the simple (single-select) searchable filter. A filter with a searchbox used to filter the results

#### How should this be manually tested?
Check storybook for searchable filters section, check new doc page for filterSearch

#### Types of changes
